frontstage/views/secure_messaging.py

```diff
@@ -42,12 +42,24 @@
 def create_message(session):
     """Handles sending of new message"""
 
+    url = app.config['RM_CASE_GET_BY_PARTY'].format(app.config['RM_CASE_SERVICE'], session['user_uuid'])
```

Comment:
still using `user_uuid` should be `party_id` as you've got @AndrewTorrance changes